### PR TITLE
remove application.<anything>. prefix

### PR DESCRIPTION
**Pull request by dorsradek**

### CODE REVIEW

File: test/expectation/.expected [hunk 1]
```diff
@@ -4,5 +4,6 @@ launch-darkly.key=secret_value
 msk.endpoint=https://msk.com
 openai.key=beepboop
 private=password
+redis.kerberos-authz.url=kerberos-authz.cache.dev.pleo.io
```
Comment (maximelaboisson) [inline test/expectation/.expected:7]: Should we add an entry to test for arrays too, like what jens wants to do? 🤔

**Review by maximelaboisson — APPROVED** [review]
LGTM

File: test/expectation/.expected (hunk 1 above)
Comment (dorsradek) [reply test/expectation/.expected:8]: it works 💣

Comment (maximelaboisson) [reply test/expectation/.expected:8]: Quality software

**Pull request merged**